src/index.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Leaderboard</title>
+</head>
+<body>
+  <h1 class="heading">Leaderboard</h1>
+  <section class="wrap">
+    <div class="heading-2">
+      <h2 class="recent-scores">Recent scores</h2>
+      <button type="button" class="refresh">Refresh</button>
+    </div>
+    <ul class="list-container">
+    </ul>
+    <div class="score-heading">
+      <h2 class="add-score">Add your score</h2>
+    </div>
+    <form action="#" class="player-info">
+      <input type="text" id="name" placeholder="Your name" />
+      <input type="text" id="score" placeholder="Your score" />
+      <div class="submit-button">
+        <button class="submit" type="submit">Submit</button>
```

Comment:
- [ ] [Optional] Kindly be noticed that the submit button should be in the right.

Your Design | WireFram 
---------------- | -------------
![Screenshot from 2022-06-16 00-54-44](https://user-images.githubusercontent.com/33752735/173955689-eb831815-29c1-4ec6-abf4-09b6797569a1.png) | ![Screenshot from 2022-06-16 00-54-54](https://user-images.githubusercontent.com/33752735/173955712-52ada79d-562a-417c-b698-f1dc23720321.png)

Reply:
Done thank you!